react/features/gifs/actions.ts

```diff
@@ -86,3 +87,16 @@ export function setGifMenuVisibility(visible: boolean) {
         visible
     };
 }
+
+/**
+ * Set visibility of the GIF drawer.
+ *
+ * @param {boolean} visible - Whether or not it should be visible.
+ * @returns {Object}
+ */
+export function setGifOverflowMenuVisibility(visible: boolean) {
```

Comment:
Why are we calling this is an overflow menu? Is it not a popover?

Reply:
It is but the component was called OverflowMenuButton. I don't have any preference here on the naming. If you want me I can rename it somehow.